frontend/src/AudioAnnotator/AudioAnnotator.tsx

```diff
@@ -446,26 +461,15 @@ class AudioAnnotator extends Component<AudioAnnotatorProps, AudioAnnotatorState>
           {}, annotation, {
             id: parseInt(newId, 10),
             annotation: this.getCurrentTag(),
-            result_comments: {
```

Comment:
Why does this property is removed ?

Reply:
I changed the `Annotation` type, now it only contains a `comment` string property, which is set in functions which call this one. So this property is not needed here anymore.